### DISCUSSION TITLE
Improve value refefencing

### DISCUSSION THREAD

**Discussion by patrickpichler** [Dynatrace/dynatrace-configuration-as-code#161] [Archive] [answered]
As of right now, referencing values is quite complicated. 

There are two ways of referencing values:
* absolute from the config root
* relative 

Since this is quite a major cause of confusion and is responsible for quite a lot 
of support work, it would make sense to redesign it.

Further discussions ins #165

**Comment by tatomir146** [comment, marked as answer]
I think we all agree we need more structure here. I would even go further and kick off discussion about current structure as IMO we could also improve this. I'm also touching how references are defined here: https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/discussions/165. Depending on what we decide, we can of course proceed referencing discussion here.

**Reply by patrickpichler** [reply]
Lets continue in the other discussion :+1: 

It seems like we cannot close this discussion, but I will lock it.
